### PR TITLE
Story1 display recent jobs

### PR DESCRIPTION
**Pull request by maxwell-01**
initial PR review ahead of trainer review

### CODE REVIEW

File: src/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const Footer = () => {
+    return (
+            <footer id="footer" className="footer-1">
```
Comment (maxwell-01) [inline src/Footer/Footer.js]: @liamlwrnc 
Please can you talk me through why you have 5 divs?
Also, the wireframe shows the footer text inline with the content whereas this footer is not.

**Review by maxwell-01 — COMMENTED** [review]
Just one comment for @liamlwrnc to take a look at on the footer

File: public/index.html [hunk 2]
```diff
@@ -1,43 +1,14 @@
...
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossOrigin="anonymous" />
+    <link rel="icon" href="magnifying-glass-tilted-right.ico" />
     <title>React App</title>
```
Comment (norayna) [inline public/index.html]: the title should be something else

File: src/Main/Main.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import JobsContainer from "../JobsContainer/JobsContainer";
+
+const Main = (props) => {
+    return (
+        <JobsContainer apiFetch={props.apiFetch}/>
+    )
+}
+
+export default Main
```
Comment (norayna) [inline src/Main/Main.js]: doesn't seem like there's any reason for this component to exist? with or without it the resulting HTML, CSS & JS generated for the website would be the same

File: src/JobsContainer/JobsContainer.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import JobTable from "../JobTable/JobTable";
+
+const JobsContainer = (props) => {
+    return (
+        <JobTable apiFetch={props.apiFetch}/>
+    )
+}
+
+export default JobsContainer
```
Comment (norayna) [inline src/JobsContainer/JobsContainer.js]: doesn't seem like there's any reason for this component to exist? with or without it the resulting HTML, CSS & JS generated for the website would be the same

File: public/index.html [hunk 5]
```diff
@@ -1,43 +1,14 @@
...
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossOrigin="anonymous" />
```
Comment (norayna) [inline public/index.html]: why do you have both bootstrap 5 and bootstrap 4? 👀

File: src/Type/Type.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+const Type = (props) => {
+
+    return (
+        <>
+            {(props.type != null) ? <span className={'badge rounded-pill bg-primary'}>{props.type}</span> : ''}
```
Comment (norayna) [inline src/Type/Type.js:5]: why is it that this component has this check but the Skill component does not have a similar check?

**Review by norayna — CHANGES_REQUESTED** [review]
looking greattt

**Review by norayna — CHANGES_REQUESTED** [review]
everything i commented on looks good now, however:

because of the bootstrap change, the job cards don't have enough padding around the content and headings now, see picture of yours and picture of wireframe:
image
image

**Review by norayna — APPROVED** [review]

**Pull request merged**